client/src/index.jsx

```diff
@@ -2,30 +2,16 @@ import React from 'react';
 import { ThemeProvider } from 'styled-components';
 import GlobalStyles from './styles/globalStyles';
 import darkTheme from './styles/theme';
+import SplashPageContainer from './splashPage/splashPageContainer';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      isEslintWorking: true,
-    };
-  }
-
-  render() {
-    const { isEslintWorking } = this.state;
-    return (
-      <ThemeProvider theme={darkTheme}>
-        <GlobalStyles />
-        <main className="container" id="app-outer-div">
-          <h1 id="title" className="header-text">Bandayd</h1>
-          <div>
-            { isEslintWorking ? 'Hello world!' : '' }
-          </div>
-        </main>
-      </ThemeProvider>
-    );
-  }
+export default function App() {
+  return (
+    <ThemeProvider theme={darkTheme}>
+      <GlobalStyles />
+      <main className="container" id="app-outer-div">
```

Comment:
@kalebal Are the class and the id being used by the styled-components at this point? If they're not being used can we take them out.

Reply:
Nope, I'll make that change